realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -118,10 +118,19 @@ public boolean shouldDeleteRealmIfMigrationNeeded() {
         return durability;
     }
 
-    public RealmProxyMediator getSchemaMediator() {
+    RealmProxyMediator getSchemaMediator() {
         return schemaMediator;
     }
 
+    /**
+     * Returns the unmodifiable {@link Set} of model classes that belong to this configuration.
+     *
+     * @return unmodifiable {@link Set} of model classes.
+     */
+    public Set<Class<? extends RealmObject>> getRealmClasses() {
```

Comment:
Given the rest of our terminology this should perhaps be `getRealmObjectClasses()` instead. What do you think?